requirements.txt

```diff
@@ -1,22 +1,23 @@
-numpy>=1.16.0
-astropy>=4.1.0
-sunpy # not currently needed
-flake8 # for code style
-black # for code style
-pytest # for testing
-pytest-astropy # for testing helpers
-pytest-cov # for testing coverage
-pre-commit # for pre-commit checks
-setuptools_scm # for version number
-sphinx # for documentation
-sphinx-automodapi # for documentation
-sphinx-changelog # for changelog in documentation
-ipython # for easier debugging
+numpy==1.24.4
+astropy==5.2.2
+sunpy==4.1.7 # not currently needed
```

Comment:
I see we're updating a lot of package versions here. Do we want to continue to have these hardcoded or just get the latest always?